doc/lua_api.md

```diff
@@ -75,40 +77,52 @@ All integer values have a range with a defined minimum and maximum.
 Integer ranges are written as [min, max] and are inclusive. E.g. the
 integer range [0, 255] contains all integers from 0 to 255,
 *including* 0 and 255.
-When the documentation does not specify a range or other restrictions,
-assume a range of [-2^37, 2^37].
 
 Some ranges in this documentation like [-2^15, 2^15-1] occur frequently,
 and may be abbreviated like so:
 
-    [int16]  = [-2^15, 2^15-1] = [-32768, 32767]
-    [int32]  = [-2^31, 2^31-1] = [-2147483648, 2147483647]
-    [uint16] = [0,     2^16-1] = [0, 65535]
-    [uint32] = [0,     2^32-1] = [0, 4294967295]
-    [uint64] = [0,     2^64-1] = [0, 18446744073709551615]
+    [s16]  = [-2^15, 2^15-1] = [-32768, 32767]
```

Comment:
could include `s8` / `u8` here

Reply:
Spelling out small numbers (lower than 1000) explicitly was intentional for readability. So no s8/u8.